### PR TITLE
Setup bare bone scripts for builds

### PR DESCRIPTION
**Pull request by abhinavdahiya**
/cc: @ericchiang

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+GOFILES:=$(shell find . -name '*.go' | grep -v -E '(./vendor)')
+GOFLAGS=
+
+all:
+	#_output/bin/$(LOCAL_OS)/binary
+	#_output/bin/linux/binary
+
+check:
+	@gofmt -l -s $(GOFILES) | read; if [ $$? == 0 ]; then gofmt -s -d $(GOFILES); exit 1; fi
+	@go vet $(shell go list ./... | grep -v '/vendor/')
```
Comment (ericchiang) [inline Makefile]: nit: `go list ./...` doesn't match vendor anymore https://golang.org/doc/go1.9#vendor-dotdotdot

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+	@./scripts/verify-gopkg.sh
+	@go test -v $(shell go list ./... | grep -v '/vendor/\|/e2e')
```
Comment (ericchiang) [inline Makefile]: Can you add `golint`?

**Review by ericchiang — COMMENTED** [review]
Just a couple nits, other than that lgtm

Once we get some Go code in here, please setup travis https://github.com/coreos/matchbox/blob/master/.travis.yml

File: Makefile (hunk 1 above)
Comment (abhinavdahiya) [reply Makefile]: fixed this

File: Makefile (hunk 2 above)
Comment (abhinavdahiya) [reply Makefile]: added golint

**Review by ericchiang — APPROVED** [review]
Lgtm. We'll see if it all works when we setup Travis :)

**Pull request merged**